src/components/Composer.vue

```diff
@@ -1121,6 +1121,13 @@ export default {
 			this.saveDraftDebounced()
 		},
 		async onSend(_, force = false) {
+			const selectedAccount = this.$store.getters.getAccount(this.selectedAlias.id)
```

Comment:
does this work with aliases too? I think `getAccount` will only return an object if the id references an account (the main address), not an alias

Reply:
https://github.com/nextcloud/mail/blob/87c3ac8db07a365b358f71e8a644b584a68289d4/src/components/Composer.vue#L654-L658
selectedAlias.id here is always the account id